.checks.yml

```diff
@@ -0,0 +1,2 @@
+- bandit
+- flake8
```

Comment:
This is a pretty generic file name also.. ;)

Reply:
haha, feel free to suggest another one. I think I will also add support for `.github/checks.yml` :P

Reply:
@codingjoe I agree with @blueyed and suggest `.fussyfox.yml` and `.github/fussyfox.yml` (or even TOML files instead of YAML, but that's another story). That's less confusing for those projects that have all the other dotfiles.